setup.cfg

```diff
@@ -7,9 +7,17 @@ packages = mbtools
 install_requires =
     lxml==4.7.1
     beautifulsoup4==4.10.0
+    boto3==1.21.17
+    python-magic==0.4.25
+    
 
 [options.extras_require]
 test =
     flake8
     pytest
     pytest-cov
+    pytest-mock
+
+[options.entry_points]
+console_scripts =
+    copy_resources_s3 = mbtools.copy_resources_s3:main
```

Comment:
Since it's more common for tools / script names to use `-` instead of `_`, I'd suggest using `copy-resources-s3 = mbtools.copy_resources_s3:main` instead.